### DISCUSSION TITLE
v5.2.0

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16292] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.2.0'>v5.2.0</a>.</em>

**Comment by lubber-de** [comment]
Thanks!
The 8.2 Windows releases are missing, could you please add them as well?

**Reply by Jeckerson** [reply]
@lubber-de 8.2 win .dlls might appear in future releases as there are still missing dependencies in Windows that are required for Phalcon compilation.

See here
https://github.com/phalcon/cphalcon/blob/35431ac12b8e53a62959a1916758ef0e0ee06356/.github/workflows/main.yml#L111

**Reply by lubber-de** [reply]
Ok, thanks for clarification

**Reply by lubber-de** [reply]
Please see https://github.com/phalcon/cphalcon/issues/16318#issuecomment-1590522445

apachelounge got a c developer who fixes and maintains tons of php extension to get them working on PHP 8.2+ and provides builds
